genesyscloud/outbound_attempt_limit/resource_genesyscloud_outbound_attempt_limit.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 const (
-	ResourceType = "genesyscloud_outbound_attemptlimit"
+	ResourceType = "genesyscloud_outbound_attempt_limit"
```

Comment:
This was what caused the tests to fail. It should be "attempt_limit" instead of "attemptlimit". 

The `SetRegistrar` function still used the correct string which is why it made no difference to the docs